File: FoodRescueFE/src/app/models/CreateFoodDto.ts
export interface CreateFoodDto {
  /** Name of the food item */
  readonly foodName: string;

  /** Number of servings available */
  readonly quantity: number;

  /** Expiry time (ISO string from backend DateTime) */
  readonly expiryTime: string;

  /** Location of the restaurant */
  readonly location: string;
}

File: FoodRescue Backend/DTOs/CreateFoodDto.cs
namespace FoodRescue_Backend.DTOs
{
    public class CreateFoodDto
    {
        public string FoodName { get; set; }
        public int Quantity { get; set; }
        public DateTime ExpiryTime { get; set; }
        public string Location { get; set; }
    }
}
